### PR TITLE
Stop loading custom diff tools during tests

### PR DESCRIPTION
**Pull request by RussKie**

### CODE REVIEW

File: IntegrationTests/UI.IntegrationTests/CommandsDialogs/FormBrowse.LeftPanelTests.Remotes.cs [hunk 1]
```diff
@@ -20,25 +20,34 @@ public class FormBrowse_LeftPanel_RemotesTests
...
         // Created once for each test
         private GitUICommands _commands;
         private IConfigFileRemoteSettingsManager _remotesManager;
-        private bool _originalShowAuthorAvatarColumn;
         private static readonly string[] RemoteNames = new[] { "remote1", "remote5", "remote3", "remote4", "remote2" };
 
         [OneTimeSetUp]
         public void SetUpFixture()
         {
+            // Remeber the current settings...
```
Comment (gerhardol) [inline IntegrationTests/UI.IntegrationTests/CommandsDialogs/FormBrowse.LeftPanelTests.Remotes.cs:35]: nit pick, some other occurrences too
```suggestion
            // Remember the current settings...
```

File: IntegrationTests/UI.IntegrationTests/CommandsDialogs/FormBrowse.LeftPanelTests.ReorderNodes.cs [hunk 2]
```diff
@@ -21,17 +21,26 @@ public class FormBrowse_LeftPanel_ReorderNodesTest
...
+        private bool _showAvailableDiffTools;
+
         private List<bool> _originalRepoObjectsTreeShow = new List<bool>();
 
         private GitModuleTestHelper _repo1;
 
         [OneTimeSetUp]
         public void SetUpFixture()
         {
+            // Remeber the current settings...
```
Comment (gerhardol) [inline IntegrationTests/UI.IntegrationTests/CommandsDialogs/FormBrowse.LeftPanelTests.ReorderNodes.cs:36]:
```suggestion
            // Remember the current settings...
```

**Review by gerhardol — APPROVED** [review]
(have not run)

File: IntegrationTests/UI.IntegrationTests/CommandsDialogs/FormBrowse.LeftPanelTests.Remotes.cs (hunk 1 above)
Comment (RussKie) [reply IntegrationTests/UI.IntegrationTests/CommandsDialogs/FormBrowse.LeftPanelTests.Remotes.cs:35]: Thank you... tired eyes

**Review by mstv — APPROVED** [review]

**Pull request merged**